test/run

```diff
@@ -293,6 +294,11 @@ function redact_instance_status {
   grep -v '^Instance status: '
 }
 
+function redact_debug_messages {
+  grep -v '^:: '
+}
```

Comment:
Why is this needed? Tests don’t run in debug mode, do they?

Reply:
For your reference, plz find below the definition of `log_debug`:
```
def log_debug(message):
    print('::', message)
```